mmdet/datasets/pipelines/transforms.py

```diff
@@ -1215,7 +1215,11 @@ def __call__(self, results):
                 raise NotImplementedError(
                     'Albu only supports BitMap masks now')
             ori_masks = results['masks']
-            results['masks'] = results['masks'].masks
+            masks_list = []
+            bitmapmasks = results['masks'].masks
+            for i in range(bitmapmasks.shape[0]):
+                masks_list += [bitmapmasks[i,:,:]]
+            results['masks'] = masks_list 
```

Comment:
Can be simplified by [`np.vsplit`](https://numpy.org/doc/stable/reference/generated/numpy.vsplit.html)